environments/template/group_vars/template.yml

```diff
@@ -81,6 +81,132 @@ tls_star_cert: star.{{ base_domain }}.pem
 tls_star_cert_key: star.{{ base_domain }}.key
 tls_ca: star.{{ base_domain }}_ca.pem
 
+springboot_service_to_deploy: all
+springboot_gui_services:
+  - name: manage
+    enabled: true
+    version: "{{ manage_gui_version }}"
+  - name: teams
+    enabled: true
+    version: "{{ teams_gui_version }}"
+  - name: pdp
+    enabled: true
+    version: "{{ pdp_gui_version }}"
+  - name: attribute-aggregation
+    enabled: true
+    version: "{{ attribute_aggregation_gui_version }}"
+  - name: dashboard
+    enabled: "{{ dashboard_install }}"
```

Comment:
I think that this is not necessary anymore.

Reply:
You mean the `dashboard` entry?

Reply:
I deleted the dashboard entry.